mac-pro-51-configurations/configs/mbpfan-ecomode1.conf

```diff
@@ -0,0 +1,44 @@
+[general]
+# see https://ineed.coffee/3838/a-beginners-tutorial-for-mbpfan-under-ubuntu for the values
+# 
+# mbpfan will load the max / min speed of from the files produced by the applesmc driver. If these files are not found it will set all fans to the default of min_speed = 2000 and max_speed = 6200
+# by setting the values for the speeds in this config it will override whatever it finds in:
+# /sys/devices/platform/applesmc.768/fan*_min
+# /sys/devices/platform/applesmc.768/fan*_max
+# or the defaults.
+#
+# multiple fans can be configured by using the config key of min_fan*_speed and max_fan*_speed
+# the number used will correlate to the file number of the fan in the applesmc driver that are used to control the fan speed.
+#
+#min_fan1_speed = 2000	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+#max_fan1_speed = 6200	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+
+
+
+min_fan1_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan1_speed = 2000	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+min_fan2_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan2_speed = 1500	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+min_fan3_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan3_speed = 1500	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+min_fan4_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan4_speed = 1500	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+min_fan5_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan5_speed = 3000	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
+
+min_fan6_speed = 900	# put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
+max_fan6_speed = 3000	# put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
```

Comment:
I think it is reasonable to add a config for Mac Pros although we don't need 6 of them.  Is there something different about Mac Pros that require this config that we could just change the defaults that would accommodate all devices?  Ideally users shouldn't need to think about these values.

Reply:
really you don’t imagine the heating inside just produced if you do anything …. and I overclock my CPU yeah ECC RAM so no bit flip. …. you see so it’s not négligeable… and I have not the key to open radiators for change thermal pasta so I think other peoples have need too this configuration, no ?

Reply:
Why do we need six configs?

Reply:
me because the mac pro classic have 6 fans and if you turn off 5 of this ... and I have never change the thermal pasta' etc ... it's old material and he need that to not burn(up easy to **70°C** if I don't use `mbpfan`) 

many thanks, `mbpfan` save my `Apple` computer `on Linux`